wondrous-app/components/Common/TaskSubtask/TaskSubtaskList/styles.tsx

```diff
@@ -0,0 +1,189 @@
+import { Typography } from '@mui/material';
+import { Button } from 'components/Common/button';
+import { SafeImage } from 'components/Common/Image';
+import { ToggleBoardPrivacyIcon } from 'components/Common/PrivateBoardIcon';
+import FilterStatusIcon from 'components/Icons/filterStatusIcon.svg';
+import styled from 'styled-components';
+
+export const TaskSubtaskWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 18px;
+  margin-top: 18px;
+`;
+
+export const TaskSubtaskItemWrapper = styled.div`
+  padding: 14px;
+  background: #1d1d1d;
+  border-radius: 2px;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+export const TaskSubtaskItemHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const TaskSubtaskItemContent = styled.div`
+  display: flex;
+  gap: 8px;
+`;
+
+export const TaskSubtaskImageWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  > * {
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+  }
+  min-width: 28px;
+`;
+
+export const TaskSubtaskRewardWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 8px;
+  min-width: fit-content;
+  border-radius: 35px;
+  background: ${({ theme }) => theme.palette.grey85};
+  height: 28px;
+  padding: 0 8px;
+`;
+
+export const TaskSubtaskRewardIcon = styled(SafeImage)`
+  && {
+    width: 13px;
+    height: 13px;
+    img[style] {
+      width: 100% !important;
+      height: 100% !important;
+    }
+  }
+`;
+
+export const TaskSubtaskRewardAmount = styled(Typography)`
+  && {
+    font-size: 13px;
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    text-transform: uppercase;
+    color: ${({ theme }) => theme.palette.white};
+  }
+`;
+
+export const TaskSubtaskPrivacyIcon = styled(ToggleBoardPrivacyIcon)`
+  && {
+    height: 28px;
+    background: #2d2d2d;
+    border-radius: 6px;
+    width: 28px;
+    min-width: 28px;
+    margin: 0;
+    svg {
+      path {
+        stroke: ${({ theme }) => theme.palette.white};
+      }
+    }
+  }
+`;
+
+export const TaskSubtaskSafeImage = styled(SafeImage)``;
```

Comment:
if we don't override the SafeImage I think we can import SafeImage directly to TaskSubtaskList